py/desitarget/test/test_htm.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from pkg_resources import resource_filename
+import numpy as np
+
+from desitarget import htm
+
+class TestHTM(unittest.TestCase):
+
+    def setUp(self):
+        self.ra = np.array([ 287.87029928,  223.95814592,  328.99793884,  127.24251626])
+        self.dec = np.array([ 16.22908364, -53.49629596,  24.40112077, -23.14789754])
+        self.id = np.array(['N023330222213310122202','S233100021230000131322','N031100100010011323332','S131112210233032311131'],dtype='<U22')
+        self.intid = np.array([14015193368226, 12043250829178, 14104942108414, 10813196528989])
```

Comment:
Would be reassuring to add tests for special cases to make sure this code handles edge cases the same as the official HTM definition.  e.g.
  * points on vertices:
    * north and south poles
    * ra=0,90,180,270 dec=0
    * at "inner" vertices at a higher depth
  * points on edges, e.g. ra=0, dec=10